sys-boot/udk/files/udk-workspace.template

```diff
@@ -0,0 +1,62 @@
+#!/bin/bash
```

Comment:
Honestly, I don't really understand why you are using bash and at the same time limiting yourself to ugly POSIX-ish syntax (i.e. minimal `[ ... ]`).

Reply:
Oh yes, I hate this syntax too. But even if this script could be called directly, it must be sourced to work. So I cannot know which shell will really execute it.

Reply:
Indeed. But then, you are using `BASH_SOURCE` in a few places. Though I guess it only really matters when calling the script directly, correct?

Reply:
The way I understood it is that BASH_SOURCE[0] contains the name of the script, be it called directly or sourced, on the contrary to $0… The values are compared to know if script is sourced.

Reply:
I just meant that it works only in bash ;-).

Reply:
Oops! Correct… What would you suggest?

Reply:
Regarding comparison, if this is not bash, the script will not detect that it is not sourced. But well, this is not a big problem. The other 2 places are used to display the name of the script to end user, for a help on script usage.

Reply:
Maybe test if -z BASH_SOURCE[0], then set it to "udk-workspace"?

Reply:
I think it's kinda fine… as long as the `[0]` part doesn't trigger syntax error. Maybe replace it with plain `${BASH_SOURCE}` that should be equivalent.